aten/src/ATen/native/native_functions.yaml

```diff
@@ -3601,12 +3601,17 @@
   dispatch:
     CPU: quantize_per_channel_cpu
 
-- func: dequantize(Tensor self) -> Tensor
+- func: dequantize.self(Tensor self) -> Tensor
   use_c10_dispatcher: full
   variants: function, method
   dispatch:
     QuantizedCPU: dequantize_quant
 
+- func: dequantize.tensors(Tensor[] tensors) -> Tensor[]
```

Comment:
Why is ListConstruct special? What about other structures like Tuple or Dict of tensors?

Reply:
we'll need to handle those as well if they come up in real use cases, but by far I didn't see that. Reason we have lists are mostly because we have quantized ops like cat that takes list as input. if we have quantized ops that takes Dict or Tuple as inputs we'll probably need to add support for these other types as well